client/views/components/generic/input-text/InputText.jsx

```diff
@@ -93,7 +93,6 @@ class InputText extends Component {
         <input
           type="text"
           disabled={readOnly}
-          placeholder={placeholder}
```

Comment:
we still want a placeholder, but just not the default value! it can be an empty string from now coming from gennyform